RELEASE.txt

```diff
@@ -103,42 +91,8 @@ access), specifically to:
         (where ``upstream`` is the name of the
         ``github.com:scikit-image/scikit-image`` repository.)
 
-
-    4. Build the package wheels (pre-compiled binaries) for various platforms:
-
-        - Clone https://github.com/scikit-image/scikit-image-wheels.
-        - If making a patch release, switch to the appropriate v<major>.<minor>.x
-          branch. Otherwise, create that branch.
-        - Update its ``.travis.yml`` file so that ``BUILD_COMMIT`` points to this
-          release tag (e.g. ``v0.14.0``).
-        - Update its ``.appveyor.yml`` file so that ``BUILD_COMMIT`` also points to
-          this release tag.
-        - Commit and push.
-        - Monitor the builds at
-            https://ci.appveyor.com/project/scikit-image/scikit-image-wheels and
-            https://travis-ci.org/scikit-image/scikit-image-wheels/branches
-        - If they fail, fix them through self-merging pull requests.
-            - Remove any build with none-supported Python versions if necessary.
-            - Add builds for new python version if necessary.
-        - When all is green, spot-check a couple of builds to make sure that the
-          wheels have correctly uploaded. You should see a message like:
-          ``python -m wheelhouse_uploader upload --local-folder``, with output:
-          ``Wheelhouse successfully published at:
-            http://[crazyrandomstring].rackcdn.com``
-        - Open that URL and check for the wheels as
-          ``scikit_image-<versiontag>-<platforminfo>.whl``
-
-- If you have performed a release candidate, redo the previous steps with the
-  final release.
-- Spot-check a couple of builds to make sure that the
-  wheels have correctly uploaded. You should see a message like:
-  ``python -m wheelhouse_uploader upload --local-folder``, with output:
-  ``Wheelhouse successfully published at:
-    http://[crazyrandomstring].rackcdn.com``
-- Open that URL and check for the wheels as
-  ``scikit_image-<versiontag>-<platforminfo>.whl``
-
-**Do not tag until the builds are all green.**
+    4. Iterate until the CI building and testing of the wheels is to your
```

Comment:
Actualy if you want to test the wheels prior to release (as a release candidate) you will need to push the tag `cibuildwheels` to only trigger the build wheels part. If you push the version tag as `vx.y.z.` it will trigger the release process.

Reply:
Good point thanks.